css/index.css

```diff
@@ -359,86 +350,91 @@ header .logo {
     header {
         flex-flow: column-reverse;
         align-items: center;
-        width: 100%;
-    }
-
-    header .logo {
-        margin: 20px -50px 0px 0px;
     }
 
     header nav {
-        flex-flow: column nowrap;
-        justify-content: space-evenly;
-        margin-top: 10px;
+        justify-content: space-between;
         width: 100%;
-
     }
 
-    header nav a {
-        padding: 20px;
-        width: 100%;
-        border-bottom: 1px solid lightgray;
-        border-top: 1px solid lightgray;
-        text-align: center;
+    header img {
+        margin: 20px;
+        min-width: 21%;
     }
 
     .left {
-        margin-right: 45px;
-        margin-top: -40px;
-    }
-
-    .description {
-        flex-flow: column wrap;
-        margin-bottom: 20px;
+        margin: 20px 0px 58px;
+        padding: 0px;
+        font-size: 13px;
     }
+}
 
```

Comment:
it is great to let people know what this media is for such as putting a comment say mobile view